### DISCUSSION TITLE
How to Synchronize Local Beatmaps and Replays Across Multiple Devices

### DISCUSSION THREAD

**Discussion by Romc3** [ppy/osu#29255] [Q&A] [answered]
I have a laptop and a desktop PC, both installed osu!lazer. However, the process of synchronizing my downloaded beatmaps and replays between these devices is tedious. Currently, I manually navigate through each beatmap's editor, export it, and then import it onto the other device. As my collection of beatmaps grows, this approach becomes increasingly time-consuming and prone to errors.

Are there a more efficient method to export all my local beatmaps, replays, and collection tags in a single process?

**Comment by SupDos** [comment, marked as answer]
Settings > Open osu folder > copy the `files` folder and the `client.realm` file to your other PC (the other files contain things like game settings and your login information - you can copy these as well if you want)

There's been a few discussions here about syncing things automatically, for example https://github.com/ppy/osu/discussions/27744, if you'd like to look into that as well
